.github/workflows/bazel_example.yml

```diff
@@ -2,9 +2,9 @@ name: Bazel Build
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, v2-dev ]
```

Comment:
We'd want to remove this before merging into main, yes?

Reply:
Now I'm thinking we want to keep it, since we're keeping the `v2-dev` branch even after this is merged.

Reply:
yeah I think we should keep it if we are keeping the branch.